plugins/push/frontend/public/javascripts/countly.views.component.common.js

```diff
@@ -503,6 +497,7 @@
                 defaultLocalizationValidationErrors: [],
                 selectionRange: null,
                 mutationObserver: null,
+                userPropertyEvents: new Map(),
```

Comment:
I hope this works fine in IE11. Map itself is supported apparently, but may be limited in terms of functionality.

Reply:
yes, the support is limited. however, I've only used the green functionalities marked from our caniuse reference guide. It's worth giving a try.